README.md

```diff
@@ -52,10 +52,7 @@ Please see the [TGI specific documentation](text-generation-inference) on how to
 `optimum-tpu` provides an optional support of JetStream Pytorch engine inside of TGI. This support can be installed using the dedicated command:
 
 ```shell
-pip install "optimum-tpu[jetstream-pt]" \
-  -f https://storage.googleapis.com/jax-releases/jax_nightly_releases.html \
-  -f https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html \
-  -f https://storage.googleapis.com/libtpu-releases/index.html
```

Comment:
I would keep it the "expended" way for easy getting started. The `make` one implies one has cloned the repo beforehands

Reply:
well, the problem is that this does not work anymore. Jetstream Pytorch is not on pypi or any indexed pypi repository. I tried adding a pinned version from github before, but it was a mess, because it installed some git submodules as packages, but since it was doing it in a temporary directory, that could get deleted by pip itself. That was quite annoying, so I ended up adding a script to clone and install the dependency manually. I could reference the script or provide a cli to install the dependency maybe. What do you think?